nxlog.sls

```diff
@@ -1,5 +1,5 @@
 nxlog:
-  {% for version in ['2.10.2102',
+  {% for version in ['2.10.2150', '2.10.2102',
```

Comment:
The older versions are causing a 404 error. So, unless you can find the URL to the older installer files, we'll probably need to remove the older ones. If that's the case, there's no point in having all the jinja code. But, I'm fine if you leave it in.

Reply:
Done